src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.TransactionDto;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitService {
+    void applyOperationOnFruitDto(List<TransactionDto> transactionDtos);
+
+    Map<String, Long> getDefaultReport();
```

Comment:
SRP. this interface is responsible for two things - applying changes on db and creating report. I think you should create report either in separate interface, or write to report file directly from db